front/migrations/db/migration_119.sql

```diff
@@ -0,0 +1,2 @@
+-- Migration created on Nov 21, 2024
+ALTER TABLE "public"."agent_conversation_include_file_actions" ADD COLUMN "tokensCount" INTEGER;
```

Comment:
We are storing the action ?
Is it mandatory ? (we'll do it at each message, no ?)

Reply:
Actions are always stored (except list_files) so that we can render the conversation and what happened

Reply:
Sorry, Indeed, I didn't carefully read that we were talking about include_file here (I was thinking of list_files).